server/conf/routes

```diff
@@ -76,9 +76,10 @@ GET     /admin/tiGroups/:id/edit           controllers.admin.TrustedIntermediary
 POST    /admin/tiGroups/:id/addTi          controllers.admin.TrustedIntermediaryManagementController.addIntermediary(id: Long, request: Request)
 POST    /admin/tiGroups/:id/removeTi       controllers.admin.TrustedIntermediaryManagementController.removeIntermediary(id: Long, request: Request)
 
-GET     /admin/tiDash                      controllers.ti.TrustedIntermediaryController.dashboard(request: Request,  search: java.util.Optional[String], page: java.util.Optional[Integer])
-POST    /admin/tiGroups/:id/addApplicant   controllers.ti.TrustedIntermediaryController.addApplicant(id: Long, request: Request)
 
+GET     /admin/tiDash                      controllers.ti.TrustedIntermediaryController.dashboard(request: Request,  search: java.util.Optional[String], searchDate: java.util.Optional[String],  page: java.util.Optional[Integer])
+POST    /admin/tiGroups/:id/addApplicant   controllers.ti.TrustedIntermediaryController.addApplicant(id: Long, request: Request)
+POST     /admin/tiGroups/:id/updateDateOfBirth/:applicantId controllers.ti.TrustedIntermediaryController.updateDateOfBirth(id: Long, applicantId: Long, request: Request)
```

Comment:
suggestion (blocking): fix spacing here to be consistent with the rest of this file